workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/DiamondWithTokenScenarioTests.kt

```diff
@@ -1,6 +1,7 @@
 package com.r3.corda.lib.tokens.workflows
 
 import com.r3.corda.lib.tokens.contracts.states.NonFungibleToken
+import com.r3.corda.lib.tokens.money.USD
```

Comment:
this import is not needed I believe

Reply:
Should I remove this in a seperate PR? It's related to DiamondWithTokenScenarioTests?